website/content/en/docs/config/vmtype/krunkit.md

```diff
@@ -31,59 +31,94 @@ go build -o <PREFIX>/libexec/lima/lima-driver-krunkit ./cmd/lima-driver-krunkit/
 limactl info   # "vmTypes" should include "krunkit"
 ```
 
-
 ## Quick start
 
-- Non‑GPU (general workloads)
+You can run AI models either:
+- With containers (fast to get started; any distro works), or
+- Without containers (choose Fedora and ensure a writable mount; build `llama.cpp` from source).
+
+Before running, install a small model on the host so examples can run quickly. We’ll use `Qwen3‑1.7B GGUF`:
+
+```bash
+mkdir -p models
+curl -LO --output-dir models 'https://huggingface.co/Qwen/Qwen3-1.7B-GGUF/resolve/main/Qwen3-1.7B-Q8_0.gguf'
+```
+
+### 1) Run models using containers (fastest)
+
+Start a krunkit VM with the default Lima template:
+
+{{< tabpane text=true >}}
+{{% tab header="CLI" %}}
+```bash
+limactl start --vm-type=krunkit
+limactl shell default
+```
+{{% /tab %}}
+{{< /tabpane >}}
+
+Then inside the VM:
+
 ```bash
-limactl start default --vm-type=krunkit
+nerdctl run --rm -ti \
+  --device /dev/dri \
+  -v $(pwd)/models:/models \
+  quay.io/slopezpa/fedora-vgpu-llama
```

Comment:
Link https://sinrega.org/2024-03-06-enabling-containers-gpu-macos/ as the reference